deco-sites/otica-isabela
# fix product review according to the needs

## What is this contribution about?

Fix ratings different than original website.

## How to test it?

Go to a PDP which has Reviews and check the number of stars.
The number must be equal to the original website.

## Extra info
<img width="1206" alt="image" src="https://github.com/deco-sites/otica-isabela/assets/47363947/67e5ffed-b030-4f88-a32a-51950b0d41ba">
